docs/source/contents.rst

```diff
@@ -0,0 +1,22 @@
+.. toctree::
```

Comment:
Replace the content of this file with:

```
.. toctree::
   :hidden:
   :glob:
   :titlesonly:

   index
   quickstart/quickstart
   connecting/connecting
   queries/queries
   data-types/data-types
   load-balancing/load-balancing
   retry-policy/retry-policy
   metrics/metrics
   logging/logging
   tracing/tracing
```

Then, for every referenced file (e.g. quickstart/quickstart.md, connecting/connecting.md, ...) add the next code snippet (eval_rst) at the end of every file:

![image](https://user-images.githubusercontent.com/9107969/114444742-1e0e8f80-9bc7-11eb-9f87-19852d620c10.png)

This is saying:  the main file ``quickstart/quickstart.md`` includes all the subpages defined in the folder quickstart.md

If you want to display them in a different order, you could also define file per file in the TocTree as follows:

![image](https://user-images.githubusercontent.com/9107969/114445219-bc025a00-9bc7-11eb-8acc-5ecc1c3eb9b9.png)

The resulting sidebar will show the pages and subpages grouped as expected:

![image](https://user-images.githubusercontent.com/9107969/114445364-e6541780-9bc7-11eb-9e10-d8993d08fdfc.png)